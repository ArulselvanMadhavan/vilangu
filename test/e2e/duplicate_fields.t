class A {
  int a, b;
  int b;
}

int main() {
  A a;
}