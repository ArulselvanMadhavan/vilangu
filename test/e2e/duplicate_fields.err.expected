./test/e2e/duplicate_fields.t:2:2 - 2:11: field b is already present in class A
